features/FEATURE_LWIP/lwip-interface/emac_lwip_l2_bridge.cpp

```diff
@@ -0,0 +1,413 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2016 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "emac_lwip_l2_bridge.h"
+#include "string.h"
+#include "mbed.h"
```

Comment:
Can this be replaced by header files that are required here? Rather than one header file `mbed.h`

Reply:
done